.github/config.yml

```diff
@@ -1,2 +1,5 @@
+chronographer:
+  enforce_name:
+    suffix: .rst
```

Comment:
We could do this through pre-commit as well, check out how pytest does this

Reply:
Well, the bot does a generic thing and it belongs there...